### PR TITLE
Refactors inventory items code to allow to play custom sounds, adds custom sounds to pill bottles and patch packs

### PR DESCRIPTION
**Pull request by Badoniondev**
This is my first time ever contributing to Paradise, if i have committed unholy mistakes of spaghetti coding, please let me know.

## What Does This PR Do
Refactors the storage_base.dm to allow for storage items (backpacks, medkits, duffelbags, anything that can contain items) to play custom sounds when opened. Pill bottles and patch packs now have their own sounds. Also fixes a previously almost unnoticeable bug where opening containers would play the same sound twice (as in, it would be just a little louder than normal)

Sounds were taken from Freesound.com and edited and resampled to 45kbps by yours truly. Sounds are licenced under [CC0](https://creativecommons.org/publicdomain/zero/1.0/)

## Why It's Good For The Game
It wouldn't make sense to hear the same ruffling sound from backpacks when opening pill boxes and patch packs. This PR aims to change that, while allowing for future PRs to add in sounds to every other inventory item should the need arise.

## Images of changes
https://github.com/ParadiseSS13/Paradise/assets/55188452/bf7903c7-6b17-4cbd-9b79-d7b08537077b

## Testing
Spawned in a pill bottle and a patch pack. On pickup and on opening of them, the correct sounds played.

## Changelog
:cl:
fix: Fixed a bug where opening inventories would play their sound twice
soundadd: Pill boxes and patch packs now have their own interaction sounds
/:cl:

### CODE REVIEW

**Review by JimKil3 — APPROVED** [review]
The code changes look good to me. I'm not too experienced with copyright stuff, so maybe double-check that with someone in #coding_chat on the discord. Also, for your changelog, you don't need to specify a name - it'll default to your github username, I believe.

**Comment by Badoniondev** [comment]
I noticed picking up a box produces both the ruffling sound and the box sound so i had to check if it was my PR that was causing it, which isn't the case after testing it through a clean download of the branch. Removing draft status.

**Comment by Badoniondev** [comment]
As for copyright, the only limitation to CC0 sounds are that you can't enforce a stricter licence on it or claim you are the author of the full sound. editing and publishing work based off CC0 is fine, even for commercial purposes (I avoided NC-Non commercial licence, since donations might be considered "commercial use"). Credits are not required on CC0 sounds but i can credit them in this PR if it's needed.

**Comment by S34NW** [comment]
> As for copyright, the only limitation to CC0 sounds are that you can't enforce a stricter licence on it(Which means the sounds i edited are also under CC0) or claim you are the author of the full sound. editing and publishing work based off CC0 is fine, even for commercial purposes (I avoided NC-Non commercial licence, since donations might be considered "commercial use"). Credits are not required on CC0 sounds but i can credit them in this PR if it's needed.

Sound assets in this codebase are licenced under Creative Commons 3.0 BY-SA unless otherwise stated. If you are specifying a separate licence for these then it needs to be laid out in a separate folder with a file indicating the licence.

**Comment by S34NW** [comment]
User has linked their account

**Comment by Badoniondev** [comment]
I moved the sounds to an appropriate folder and i included a txt with the license. Is that good?

File: code/game/sound.dm [hunk 1]
```diff
@@ -189,6 +189,10 @@ falloff_distance - Distance at which falloff begins. Sound is at peak volume (in
 /proc/get_sfx(soundin)
 	if(istext(soundin))
 		switch(soundin)
+			if("patchpack")
+				soundin = pick('sound/effects/patchpack1.ogg', 'sound/effects/patchpack2.ogg', 'sound/effects/patchpack3.ogg', 'sound/effects/patchpack4.ogg', 'sound/effects/patchpack5.ogg')
+			if("pillbottle")
+				soundin = pick('sound/effects/pillbottle1.ogg', 'sound/effects/pillbottle2.ogg', 'sound/effects/pillbottle3.ogg', 'sound/effects/pillbottle4.ogg')
```
Comment (S34NW) [inline code/game/sound.dm:195]: paths are no longer valid

**Review by S34NW — CHANGES_REQUESTED** [review]

File: code/game/sound.dm (hunk 1 above)
Comment (Badoniondev) [reply code/game/sound.dm:195]: Slipped my mind, thanks for reminding me.

**Pull request merged**